.editorconfig

```diff
@@ -10,7 +10,7 @@ charset = utf-8
 [{*.c,*.cpp,*.h,Makefile}]
 trim_trailing_whitespace = true
 insert_final_newline = true
-end_of_line = lf
+end_of_line = crlf
```

Comment:
Maybe `end_of_line` should be removed completely (or at least until `native` is supported) since it's not universal across macOS/Linux/Windows.

Reply:
I agree with you

Reply:
> @Acenotass (and anyone else) I'd recommend installing the [EditorConfig plugin](https://github.com/editorconfig/editorconfig-vscode). This will keep code tidy and ensure proper file encoding. ~I'll put in a PR shortly to add *.ini files to the UTF-8 encoding section~: [Done](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/539).

@thisiskeithb It keeps adding a new line to the code if you are typing at the end of the file and stop for 2 seconds. I think this is due to `insert_final_newline = true`. So I set those settings in VSCodium's default settings.

Reply:
Odd. Which OS are you using? I don’t have that issue on macOS or Windows.

Reply:
Windows